docs/HowToHashiCorpVault.md

```diff
@@ -0,0 +1,278 @@
+# Using HashiCorp Vault for Management of TLS Certificates
+
+## Introduction
+
+NuoDB supports TLS encryption for all processes in the domain.
+`NuoDB Admin` is responsible for propagating certificates to database processes
+To enable TLS encryption for all processes, it is necessary to configure NuoDB Admin with a set of certificates, and also configure NuoDB Command (`nuocmd`) clients to be able to communicate with `NuoDB Admin`.
+
+This document expands on [Security Model of NuoDB in Kubernetes](./HowtoTLS.md) and explains the usage of HashiCorp Vault for the management of TLS keys.
+
+HashiCorp uses sidecar containers to inject keys into another container.
+For more info, explore the blog post [Injecting Vault Secrets](https://www.hashicorp.com/blog/injecting-vault-secrets-into-kubernetes-pods-via-a-sidecar).
+
+
+> **NOTE**: For information on enabling  TLS encryption in non-Kubernetes deployments of NuoDB, see [here](http://doc.nuodb.com/Latest/Content/Nuoadmin-Configuring-TLS-Security.htm). 
```

Comment:
This is an old doc link, which redirects to 4.0.x. Can you replace this with https://doc.nuodb.com/nuodb/latest/deployment-models/physical-or-vmware-environments-with-nuodb-admin/domain-operations/enabling-tls-encryption/?

Reply:
sure!